### ISSUE TITLE
extract text is not working since PDFs were moved to AWS

### ISSUE THREAD

**Issue by caseyg** [disorientations/disorientations.org#10]
started a thread here but need to do some more testing https://forum.omeka.org/t/extracttext-for-media-hosted-on-amazons3/10994/2

> I installed @jimsafley’s module ExtractText and it worked great!
> 
> As I added to my Omeka archive, I quickly began to run out of local storage space, so I also activated @Daniel_KM’s AmazonS3 module with ArchiveRepertory — and I was able to upload a lot more files.
> 
> However, I just realized that ExtractText’s controls are disabled when it detects remote (e.g. s3) media, and no text is extracted when new remote media are added.
> 
> Is this a technical limitation? Or just a missing feature for which a workaround/modification might be possible?
> 
> I started poking around between the ExtractText and the AmazonS3 module to see if I could comment out certain lines checking for local storage, and feed in the remote media URIs … but I didn’t get it working and was in a bit over my head. (This is my first Omeka project!)

**Comment by caseyg** [comment]
@elazar Since this one is Omeka plugin related, I think I wanna try to get #20 sorted first (connecting the site to Github) so that there are some repos for the theme + plugins, etc. which we can iterate on. Right now it's starting to get kinda messy because it was all set up via installer GUIs and then I started making changes via FTP 🤣. 

Will ping when that's done, or if I get stuck!

I can also drop more notes here on the specific plugins I'm using, and how they intersect -- to hopefully save you some digging.

**Comment by elazar** [comment]
Note to self: here's [the repo](https://github.com/omeka-s-modules/ExtractText) for the ExtractText module.

**Comment by elazar** [comment]
@caseyg Reassigning this to you, to do with as you like.

We found that the issue turned out to be that pdftotext doesn't handle text contained in images, as opposed to embedded text in the PDF. I've added the appropriate package to include pdftotext to the Docker setup in this repository. 

If there's no directly related residual action to take on this issue as it was originally written, I'd suggest closing this issue and opening a new one more focused on adding actual OCR capabilities to Omeka.
